templates/page.tmpl

```diff
@@ -55,17 +55,45 @@
         <figure class="lightmode" id="justified-logo">
             <img alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome2.svg">
         </figure>
-
-        <nav id="navbar" class="container active-menu-<TMPL_VAR NAME="title">">
-            <a class="menu-about" alt="home" href="<TMPL_VAR BASEURL>index.html">home</a>
-            <a class="menu-download" alt="download" href="<TMPL_VAR BASEURL>download/">download</a>
-            <a class="menu-community" alt="community" href="<TMPL_VAR BASEURL>community/">community</a>
-            <a class="menu-recipes" alt="recipes" href="<TMPL_VAR BASEURL>recipes/">recipes</a>
-            <a class="menu-screenshots" alt="screenshots" href="<TMPL_VAR BASEURL>screenshots/">screenshots</a>
-            <a class="menu-doc" alt="doc" href="<TMPL_VAR BASEURL>doc/">doc</a>
-            <a class="menu-issues" alt="bugs/issues" href="https://github.com/awesomeWM/awesome/issues">bugs/issues</a>
+    </header>
+    <header id="navbar">
+        <a class="logo" href="/">
+            <figure class="darkmode zero-margins">
+                <img id="nav-logo-dark" alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome-dark-1.svg">
+            </figure>
+            <figure class="lightmode zero-margins">
+                <img id="nav-logo-light" alt="awesome-logo" src="<TMPL_VAR BASEURL>images/awesome2.svg">
+            </figure>
```

Comment:
A `<figure>` specifies self-contained content (e.g. a diagram or a photo), usually within a body of text and accompanied by a `<figcaption>`.
It doesn't make much sense for a logo.